tasks.py

```diff
@@ -606,12 +606,13 @@ def api_docs(ctx: Context):
         "lint": "Run the linter",
     },
 )
-def docs(
+def docs(  # noqa: PLR0913
```

Comment:
We now have 6 arguments because there is now a `--clear` flag. I could create a new command to do this but this felt better. Let me know if you have an alternative.

Reply:
I think we turn off this rule for `tasks.py`
`tool.ruff.lint.per-file-ignores`

Reply:
I would add a brief description for this param.
```python
@invoke.task(
    name="docs",
    help={
        "build": "Build docs via yarn build instead of serve via yarn start. Default False.",
        "start": "Only run yarn start, do not process versions. For example if you have already run invoke docs and just want to serve docs locally for editing.",
        "lint": "Run the linter",
        "clear": "delete stuff 🤷 "
    },
)
def docs(
```

```console
$ invoke docs --help

Usage: inv[oke] [--core-opts] docs [--options] [other tasks here ...]

Docstring:
  Build documentation site, including api documentation and earlier doc versions. Note: Internet access required to download earlier versions.

Options:
  -b, --build                   Build docs via yarn build instead of serve via yarn start. Default False.
  -l, --lint                    Run the linter
  -s, --start                   Only run yarn start, do not process versions. For example if you have already run invoke docs and just want to serve docs locally for editing.
  -v STRING, --version=STRING
```

Reply:
Done.